tests/cases/standalone/common/types/interval/interval.result

```diff
@@ -264,6 +264,47 @@ SELECT TIMESTAMP '1992-09-20 11:30:00.123456' - interval_value as new_value from
 | 1980-09-07T23:17:48.111443988 |
 +-------------------------------+
 
+-- Interval shortened names
+SELECT INTERVAL '55h';
+
++--------------------------------------------------------+
+| IntervalMonthDayNano("198000000000000")                |
++--------------------------------------------------------+
+| 0 years 0 mons 0 days 55 hours 0 mins 0.000000000 secs |
++--------------------------------------------------------+
+
+SELECT INTERVAL '-2mon';
+
++---------------------------------------------------------+
+| IntervalMonthDayNano("-158456325028528675187087900672") |
++---------------------------------------------------------+
+| 0 years -2 mons 0 days 0 hours 0 mins 0.000000000 secs  |
++---------------------------------------------------------+
+
+SELECT INTERVAL '1y2w3d4h';
+
++---------------------------------------------------------+
+| IntervalMonthDayNano("950737950484766714775589781504")  |
++---------------------------------------------------------+
+| 0 years 12 mons 17 days 4 hours 0 mins 0.000000000 secs |
++---------------------------------------------------------+
```

Comment:
Why is the result not `1 years 0 mons 17 days 4 hours 0 mins 0.000000000 secs`?

Reply:
Good point! 
When I see that I also was a bit concerned, but looks like it's internal DataFusion tweaks.
Out of curiosity I ran a similar test, and it shows similar result (maybe to be a bit more compact in the layout 🤔)

```
SELECT INTERVAL '1y1w1d1h';

+--------------------------------------------------------+
| IntervalMonthDayNano("950737950318746007312203816960") |
+--------------------------------------------------------+
| 0 years 12 mons 8 days 1 hours 0 mins 0.000000000 secs |
+--------------------------------------------------------+
```

Reply:
Not relevant to this PR. It's implementation of SQPParser.